app-backend/api/src/main/scala/user/Auth0User.scala

```diff
@@ -323,32 +349,142 @@ object Auth0Service extends Config with LazyLogging {
       .flatMap { responseAsAuth0User _ }
   }
 
-  def createAnonymizedUser(
-      userName: String,
+  @JsonCodec
+  final case class ImportResponse(status: String, id: String)
+
+  @JsonCodec
+  final case class Auth0JobStatus(status: String)
+
+  def waitForJobToComplete(
```

Comment:
I tried that, this isn't a tail recursive function and I didn't put in a ton of time to try and figure out how to change it since there's an upper bound on the number of calls.

Reply:
Oh dang ok I guess because the recursion is inside a flatMap maybe? That's too bad 😢